sdk/python/tests/test_online_retrieval.py

```diff
@@ -234,3 +236,163 @@ def test_online() -> None:
 
         # Restore registry.db so that teardown works
         os.rename(store.config.registry + "_fake", store.config.registry)
+
+
+def test_online_to_df():
+    """
+    Test dataframe conversion. Make sure the response columns and rows are
+    the same order as the request.
+    """
+
+    driver_ids = [1, 2, 3]
+    customer_ids = [4, 5, 6]
+    name = "foo"
+    lon_multiply = 1.0
+    lat_multiply = 0.1
+    age_multiply = 10
+    avg_order_day_multiply = 1.0
+
+    runner = CliRunner()
+    with runner.local_repo(
+        get_example_repo("example_feature_repo_1.py"), "bigquery"
+    ) as store:
+        # Write three tables to online store
+        driver_locations_fv = store.get_feature_view(name="driver_locations")
+        customer_profile_fv = store.get_feature_view(name="customer_profile")
+        customer_driver_combined_fv = store.get_feature_view(
+            name="customer_driver_combined"
+        )
+        provider = store._get_provider()
+
+        for (d, c) in zip(driver_ids, customer_ids):
+            """
+            driver table:
+            driver driver_locations__lon  driver_locations__lat
+                1                   1.0                    0.1
+                2                   2.0                    0.2
+                3                   3.0                    0.3
+            """
+            driver_key = EntityKeyProto(
+                join_keys=["driver"], entity_values=[ValueProto(int64_val=d)]
+            )
+            provider.online_write_batch(
+                project=store.project,
+                table=driver_locations_fv,
```

Comment:
This API was updated recently, you may need to rebase and provide a `config` kwarg instead of `project`.